src/main/java/me/xmrvizzy/skyblocker/skyblock/item/ItemCooldowns.java

```diff
@@ -0,0 +1,115 @@
+package me.xmrvizzy.skyblocker.skyblock.item;
+
+import com.google.common.collect.ImmutableList;
+import me.xmrvizzy.skyblocker.config.SkyblockerConfigManager;
+import me.xmrvizzy.skyblocker.events.ClientPlayerBlockBreakEvent;
+import me.xmrvizzy.skyblocker.utils.ItemUtils;
+import net.fabricmc.fabric.api.event.player.UseItemCallback;
+import net.minecraft.block.BlockState;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.item.ItemStack;
+import net.minecraft.sound.BlockSoundGroup;
+import net.minecraft.util.Hand;
+import net.minecraft.util.TypedActionResult;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ItemCooldowns {
+    private static final String JUNGLE_AXE_ID = "JUNGLE_AXE";
+    private static final String TREECAPITATOR_ID = "TREECAPITATOR_AXE";
+    private static final String GRAPPLING_HOOK_ID = "GRAPPLING_HOOK";
+    private static final ImmutableList<String> BAT_ARMOR_IDS = ImmutableList.of("BAT_PERSON_HELMET", "BAT_PERSON_CHESTPLATE", "BAT_PERSON_LEGGINGS", "BAT_PERSON_BOOTS");
+
+    private static final Map<String, CooldownEntry> ITEM_COOLDOWNS = new HashMap<>();
+
+    public static void init() {
+        ClientPlayerBlockBreakEvent.AFTER.register(ItemCooldowns::afterBlockBreak);
+        UseItemCallback.EVENT.register(ItemCooldowns::onItemInteract);
+    }
+
+    public static void afterBlockBreak(World world, PlayerEntity player, BlockPos pos, BlockState state) {
+        if (!SkyblockerConfigManager.get().general.itemCooldown.enableItemCooldowns) return;
+
+        String usedItemId = ItemUtils.getItemId(player.getMainHandStack());
+        if (usedItemId == null) return;
+
+        if (state.getSoundGroup() == BlockSoundGroup.WOOD && state.isBurnable()) {
```

Comment:
I feel like this is not the best way. Can you do something like `state.isIn(BlockTags.LOGS)`?

Reply:
Yeah, `state.isIn(BlockTags.LOGS)` makes more sense.